src/app/filters/components/FilterGroupItem.jsx

```diff
@@ -63,35 +61,27 @@ class FilterGroupItem extends Component {
           hue={this.props.filterGroup.hue}
           onToggled={() => this.onChangeFilterGroupVisibility()}
         />
-        <input
+        <div
```

Comment:
We can't use a div here, an input is needed for bablic to translate the content.

Reply:
Sorry, this is my fault because the bablic translations only needs an input for custom layers and not by filters. Merging